src/main/java/com/bieganski/ox/PositionCalculator.java

```diff
@@ -11,7 +11,7 @@
 //TODO maybe parser?
     int calculatePosition(String userInput) {
```

Comment:
`Integer.parseInt` to metoda na przerabianie napisu w numer. Jak robimy coś podobnego, to trzymajmy konwencję nazewniczą. Przychylam się do notki TODO.